package.json

```diff
@@ -16,7 +18,7 @@
   "main": "dist/rollup-plugin-npm.cjs.js",
   "jsnext:main": "dist/rollup-plugin-npm.es6.js",
   "scripts": {
-    "test": "mocha",
+    "test": "mocha --compilers js:babel-register",
```

Comment:
I've struggled with the right way to do this. On the one hand, this means that Babel is handling modules and it may do so in a way that differs from how rollup does it. Hopefully this default export issue will be rectified in the near future (by Babel), but it is still an issue now.

Lately I've been building the test bundle with a slightly different version of my dist configuration that uses the multi-entry plugin to pull in all the tests.